routes/web.php

```diff
@@ -13,6 +14,9 @@
 |
 */
 
+
 Route::get('/', function () {
-    return view('welcome');
+    return view('home');
 });
+
+Route::get('/airline', GetAirlineController::class);
```

Comment:
```suggestion
Route::get('/airlines', GetAirlinesController::class);
```
Notice I pluralized both the prefix and the controller name (`Airline` to `Airlines`).